index.html

```diff
@@ -3,6 +3,12 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
+    />
```

Comment:
```suggestion
    <link
      rel="preload"
      as="style"
      crossorigin
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
    />
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
      />
```
preload 관련해서 적어주신것 확인했습니다. 제가 알기로는 preload는 필요한 리소스를 dom load 전에 미리 가져오기만 하는걸로 알고 있고 stylesheet까지 해야 폰트를 css로 파싱해서 사용할 수 있는 것으로 알고 있습니다.
그래서 위에처럼 해보고 다음 2가지가 맞는지 확인해보고 맞으면 이렇게 사용하면 될 것 같아요!

1. 네트워크 요청이 한번만 가는지 확인
2. DOMContentLoaded가 preload 적용했을때 더 늦어지는지 확인

Reply:
넵 확인해보겠습니다 : ) 감사합니다

Reply:
흠..말씀해주신대로 해봤는데 어쉽게도 요청이 두 번 가는 것 같습니다ㅜㅜ 다른 방법을 강구해봐야 할까요?
<img width="987" alt="스크린샷 2024-01-19 오전 2 23 46" src="https://github.com/tteokguk-please/tteokguk-please-v2/assets/61978339/72fa7af6-2b0e-456a-ab98-422243aeedec">

```html
    <link
      rel="preload"
      as="style"
      crossorigin
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
    />
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
    />
```

Reply:
```
    <link
      rel="preload"
      as="style"
      crossorigin
      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
    />
```
음 근데 다시 해보니 이것만 했을때도 저는 잘 되는것 같은데.. 잘 안되시나요??

Reply:
그게 파일은 잘 도착하는데...눈누에서 보이는 ㅊ과 제 프로젝트의 ㅊ모양이 다른거 보이시나요?
`rel=stylesheet`로 바꾸면 눈누와 동일하게 되는데 preload일때는 다르더라구요.
<img width="531" alt="스크린샷 2024-01-19 오전 2 54 50" src="https://github.com/tteokguk-please/tteokguk-please-v2/assets/61978339/3f929879-2612-43b0-8234-53b286cbfd2e">

<img width="655" alt="스크린샷 2024-01-19 오전 2 55 57" src="https://github.com/tteokguk-please/tteokguk-please-v2/assets/61978339/48b270f3-2abd-4810-a691-57719ea516b5">

Reply:
아 그러네요ㅜㅜ 우선 이대로 가고 나중에 좀 더 찾아보는걸로 하시죠..!

Reply:
알겠습니다!